packages/manager/modules/anthos/src/dashboard/routing.js

```diff
@@ -1,5 +1,5 @@
 import { ANTHOS_TENANT_ALERTER } from '../anthos.constants';
```

Comment:
**Blocker** : the changes should me made at the top of the service, in
`packages/manager/modules/anthos/src/anthos-tenants.service.js`
Within the function `getServiceInfo` :+1: